### ISSUE TITLE
Bug: clearAllMocks doesn't clear static mock of java.lang.System

### ISSUE THREAD

**Issue by vehovsky** [mockk/mockk#234]
version : 1.9

### Expected Behavior
```
clearAllMocks()
```
Should clear all static mocks.

### Current Behavior
 
```
mockkStatic("java.lang.System")
...
clearAllMocks()
```

calling `System.getProperty("any_string")` will always return empty String. Causing initialization failure in later tests that are using:

```
coEvery { .. } 
```

Because of:
```
Caused by:
java.lang.IllegalStateException: System property 'kotlinx.coroutines.scheduler.resolution.ns' has unrecognized value ''
```

**Comment by oleksiyp** [comment]
Hm. Tried to reproduce with the following code:
```
        System.setProperty("def", "ghi")
        mockkStatic("java.lang.System")
        every { System.getProperty("def") } returns "abc"
        println(System.getProperty("def"))
        clearAllMocks()
        println(System.getProperty("def"))
```

Output(besides warnings from JVM) is:
```
abc
ghi
```
Please specify which exact code sequence leads to the problem. Thanks

**Comment by vehovsky** [comment]
Hi,

hmm I can't reproduce from home. I'll try again when in work on Monday. Basically this was the case for me:

```
println(System.getProperty("undef"))

mockkStatic("java.lang.System")
every { System.getenv("SomeProperty") } returns "true"
clearAllMocks()

println(System.getProperty("undef"))
```

Where second call for "undef" returned empty string instead of null.

**Comment by vehovsky** [comment]
I have minimalistic reproducible example:

```
    data class Example(val a: String) {
        suspend fun a() {
            // logic
        }
    }

    @Test
    fun `System mock test`() {
        mockkStatic("java.lang.System")
        clearAllMocks()
        //unmockkStatic("java.lang.System")
        mockkConstructor(Example::class)
        coEvery { anyConstructed<Example>().a() } just runs
    }
```

The "unmockkStatic" can is a workaround for now. But without it you will get `java.lang.ExceptionInInitializerError` described above.

**Comment by vehovsky** [comment]
@oleksiyp can you please try?

**Comment by oleksiyp** [comment]
Tried. Actually it is not possible to fix all issues around such classes as `System`, but I fixed this particular case

**Comment by oleksiyp** [comment]
Released [1.9.1](https://github.com/mockk/mockk/releases/tag/1.9.1). Please try it out and close the ticket if everything is fine

**Comment by vehovsky** [comment]
Tested, works great. Thank you!
